crates/bevy_ecs/src/archetype.rs

```diff
@@ -740,6 +747,13 @@ impl Archetypes {
             })
     }
 
+    pub fn shrink_to_fit(&mut self) {
```

Comment:
This should probably be `pub(crate)`. There's no way for a user of `bevy_ecs` to get a `&mut Archetypes` without being unsound.